data-serving/data-service/src/controllers/case.ts

```diff
@@ -78,6 +78,7 @@ const dtoFromCase = async (storedCase: LeanDocument<CaseDocument>) => {
         delete (dto as unknown as { demographics: { ageBuckets?: [ObjectId] }}).demographics.ageBuckets;
     }
     delete dto.restrictedNotes;
+    delete dto.notes;
```

Comment:
Maybe this is a "not yet" task but we could migrate `restrictedNotes` back over to `notes`, as _all_ notes are now restricted. Then we would be able to build a single approach to exposing notes later.